static/app/components/profiling/profilingTransactionHovercard.tsx

```diff
@@ -0,0 +1,301 @@
+    sort: {
+      key: 'timestamp',
+      order: 'desc',
+    },
+  });
+
+  const functions = useFunctions({
+    project,
+    query: '',
+    selection,
+    transaction,
+    sort: '-p99',
+    functionType: 'application',
+  });
+
+  const slowestProfile = slowestProfileQuery?.data?.[0].data[0] ?? null;
+  const durationUnits = slowestProfileQuery.data?.[0].meta.units['profile.duration'];
+  const multiplier = durationUnits ? DURATION_UNITS[durationUnits] ?? 1 : 1;
+
+  const latestProfile = latestProfileQuery?.data?.[0].data[0] ?? null;
+
+  const linkToFlamechartRoute = (
+    profileId: string,
+    query?: {frameName: string; framePackage: string}
+  ) => {
+    return generateProfileFlamechartRouteWithQuery({
+      orgSlug: organization.slug,
+      projectSlug: project.slug,
+      profileId,
+      query,
+    });
+  };
+
+  return (
+    <Flex gap={space(3)} column>
+      <Flex justify="space-between">
+        <ContextDetail
+          title={t('Latest profile')}
+          isLoading={latestProfileQuery.isLoading}
+        >
+          {latestProfile ? (
+            <Link to={linkToFlamechartRoute(String(latestProfile.id))}>
+              {getShortEventId(String(latestProfile!.id))}
+            </Link>
+          ) : (
+            '-'
+          )}
+        </ContextDetail>
+
+        <ContextDetail
+          title={t('Slowest profile')}
+          isLoading={slowestProfileQuery.isLoading}
+        >
+          {slowestProfile ? (
+            <Flex gap={space(1)}>
+              <PerformanceDuration
+                milliseconds={multiplier * (slowestProfile['profile.duration'] as number)}
+                abbreviation
+              />
+              <Link to={linkToFlamechartRoute(String(slowestProfile.id))}>
+                ({getShortEventId(String(slowestProfile?.id))})
+              </Link>
+            </Flex>
+          ) : (
+            '-'
+          )}
+        </ContextDetail>
+      </Flex>
+
+      <Flex column h={125}>
+        <FunctionsMiniGrid>
+          <FunctionsMiniGridHeader>{t('Slowest app functions')}</FunctionsMiniGridHeader>
+          <FunctionsMiniGridHeader align="right">{t('P99')}</FunctionsMiniGridHeader>
+          <FunctionsMiniGridHeader align="right">{t('Count')}</FunctionsMiniGridHeader>
+
+          {functions.type === 'resolved' &&
+            functions.data.functions.map(f => {
+              const [exampleProfileIdRaw] = f.examples;
+              const exampleProfileId = exampleProfileIdRaw.replaceAll('-', '');
+              return (
+                <Fragment key={f.name}>
+                  <FunctionsMiniGridCell title={f.name}>
+                    <TextTruncate>
+                      <Link
+                        to={linkToFlamechartRoute(exampleProfileId, {
+                          frameName: f.name,
+                          framePackage: f.package,
+                        })}
+                      >
+                        {f.name}
+                      </Link>
```

Comment:
i'm not a huge fan of how we just link to the first arbitrary profile of a suspect function.

we also need to show some more relevant functions, here is an example of all the functions w/ the same name:
![image](https://user-images.githubusercontent.com/7349258/212400127-b8325e17-c379-4541-ad4f-bc74d40016c2.png)


it can be confusing for users